code/demos/Simulation.ipynb

```diff
@@ -641,22 +634,39 @@
     "        dessert.append(fit_any(true_vals))\n",
     "    return np.array(dessert)\n",
     "\n",
-    "sheet_cake = fit_all(true_params)\n",
-    "\n",
+    "sheet_cake = fit_all(true_params)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We plot a handful of these likelihoods.  Each row is the log-likelihood for a different supernova in the catalog.  The columns represent the log-likelihood in the space of $z$ and $\\mu$ (Hubble diagram-space) for that supernova if it were classified as type $Ia$, type $Ibc$ and type $II$.  The true $z$ and $\\mu$ for each are also plotted.  The title of each panel gives the true type for each supernova.  We can see the effect on the likelihoods for $z$ and $\\mu$ based on each combination of true and classified type."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
```

Comment:
Cool to have per-SN info, but we should be careful about All the Plotz All the Time.